index.bs

```diff
@@ -264,16 +266,16 @@ interface Sanitizer {
   SanitizerConfig get();
 
   // Modify a Sanitizer's lists and fields:
-  undefined allowElement(SanitizerElementWithAttributes element);
-  undefined removeElement(SanitizerElement element);
-  undefined replaceElementWithChildren(SanitizerElement element);
-  undefined allowAttribute(SanitizerAttribute attribute);
-  undefined removeAttribute(SanitizerAttribute attribute);
-  undefined setComments(boolean allow);
-  undefined setDataAttributes(boolean allow);
+  boolean allowElement(SanitizerElementWithAttributes element);
+  boolean removeElement(SanitizerElement element);
+  boolean replaceElementWithChildren(SanitizerElement element);
+  boolean allowAttribute(SanitizerAttribute attribute);
+  boolean removeAttribute(SanitizerAttribute attribute);
+  boolean setComments(boolean allow);
+  boolean setDataAttributes(boolean allow);
 
   // Remove markup that executes script. May modify multiple lists:
```

Comment:
"May modify multiple lists" applies to the methods above as well. Perhaps it would be clearer to not state anything about that.

Reply:
Done.